code/modules/uplink/uplink_item.dm

```diff
@@ -1051,6 +1052,49 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	item = /obj/item/organ/cyberimp/chest/reviver
 	cost = 8
 
+/datum/uplink_item/cyber_implants/esword	//This is a bad idea
+	name = "Arm Energy-Blade Projector"
+	desc = "An illegal cybernetic combat implant that allows the user to project a blade of lethal energy from their hand."
+	item = /obj/item/organ/cyberimp/arm/esword
+	cost = 12
+	player_minimum = 30
+
+/datum/uplink_item/cyber_implants/esword/traitor	//This is a worse idea
+	name = "Arm Energy-Blade Projector"
```

Comment:
Object oriented programming!! You don't need to specify the variables in the child if they are identical to the parent!!